triplets/community/x64-windows-static-md-release.cmake

```
@@ -0,0 +1,5 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_LIBRARY_LINKAGE static)
+
+set(VCPKG_BUILD_TYPE release)
```

```suggestion
set(VCPKG_BUILD_TYPE release)

```
We need to add the end line otherwise it will appear in red. :)